codecov.yml

```
@@ -0,0 +1,9 @@
+codecov:
+  require_ci_to_pass: false
+
+coverage:
+  status:
+    project:
+      default:
+        informational: true
+        
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)